misc/event.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+"""
+Select event ID from time and magnitude intervals.
+"""
+from __future__ import print_function
+import sys
+
+def event(baseurl='IRIS', 
+		starttime=None, 
+		endtime=None,
+		minmag=3., 
+		maxmag=10., 
+		maxnumber=10, 
+		catalog=None):
+
+	from obspy import UTCDateTime	
```

Comment:
Is there a particular reason to have the imports inside the function? I don't want to impose pep8 on you but I think their guidelines on imports and indenting make a lot of sense https://www.python.org/dev/peps/pep-0008/#imports